sw-emulator/lib/periph/src/soc_reg.rs

```diff
@@ -1102,7 +1136,15 @@ impl Bus for SocRegistersImpl {
             }
 
             INTERNAL_FW_UPDATE_RESET_START..=INTERNAL_FW_UPDATE_RESET_END => {
-                self.internal_fw_update_reset.write(size, val)
+                self.internal_fw_update_reset.write(size, val)?;
+
+                // Schedule a firmware update reset timer action.
+                self.op_reset_trigger_action = Some(
```

Comment:
Does the Verilog code also do this? If not, I'd prefer to move this hacky "special case" logic out of the emulator library to somewhere else, as we might accidentally rely on it in production code.

Reply:
A write to internal_fw_update_reset will trigger an update reset in Caliptra; this is per documented behavior.

Reply:
This is specific to the emulator. Other option is to define a emulator specific register to trigger the warm reset.

Reply:
Disabled the warm reset flow. We'll enable it on reaching design agreement.